mlflow/server/js/src/components/ExperimentView.js

```diff
@@ -679,6 +683,10 @@ class ExperimentView extends Component {
       ];
       const paramsMap = ExperimentViewUtil.toParamsMap(paramsList[index]);
       const metricsMap = ExperimentViewUtil.toMetricsMap(metricsList[index]);
+
+      /* add run name to export row */
+      row[1] = Utils.getRunName(tagsList[index]);
```

Comment:
Nice, this works but maybe we can just replace `runInfo.name` [here](https://github.com/mlflow/mlflow/pull/864/files#diff-fa151380130110b5b1f7ba277afb356aR678) with `Utils.getRunName(tagsList[index])`, rather than set the value and then overwrite it here